scripts/rollup/plugins/strip-unused-imports.js

```diff
@@ -14,8 +14,8 @@ module.exports = function stripUnusedImports(pureExternalModules) {
         // Ideally this would use a negative lookbehind: (?<!= *)
         // But this isn't supported by the Node <= 8.9.
         // So instead we try to handle the most common cases:
-        // 1. foo,bar=require("bar"),baz
-        // 2. foo;bar = require('bar');baz
+        // 1. foo,require("bar"),baz
+        // 2. foo;require('bar');baz
```

Comment:
Looking at the defined regular expressions, the first and second are not unused import cases, Introduced by this pr https://github.com/facebook/react/pull/13538, @bvaughn Can you please confirm this question?